json-guides/retry-timeout.json

```diff
@@ -101,6 +101,8 @@
             "description": [
                 "Now that you’ve seen the page loads indefinitely when there is some problem on the server end, let's apply a Timeout policy to limit the time the request is made to a transaction history and to prevent the page from loading for a long time.",
                 "The <code>@Timeout</code> annotation takes an integer value that specifies the time allowed for the request to finish. The value must be greater than or equal to <code>0</code>. <code>0</code> means no timeout configured. If the value is not specified, the default is <code>1000</code> milliseconds.",
+                "The <code>@Timeout</code> annotation can also be configure to change the default unit with the following parameters:",
```

Comment:
... can also be configure**d** with the following parameters to change the default unit.